CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 2.3.0-RC4
+
+### Bugfixes
+
+- **Home Page**
```

Comment:
Please remove the changelog entry from the changelog.md file. Could you mention it in the description of your PR?

Reply:
reverted changelog commits, and updated into the ticket description